src/numerics/mesh_utils.cpp

```diff
@@ -170,16 +170,23 @@ std::shared_ptr<mfem::ParMesh> buildCuboidMesh(int elements_in_x, int elements_i
 
 namespace mesh {
 
-void defineInputFileSchema(std::shared_ptr<axom::inlet::SchemaCreator> schema_creator)
+void defineInputFileSchema(axom::inlet::Table& schema_creator)
 {
   // mesh path
-  schema_creator->addString("mesh", "Path to Mesh file")->required(true);
+  schema_creator.addString("mesh", "Path to Mesh file")->required(true);
 
   // Refinement levels
-  schema_creator->addInt("ser_ref_levels", "Number of times to refine the mesh uniformly in serial.")->defaultValue(0);
-  schema_creator->addInt("par_ref_levels", "Number of times to refine the mesh uniformly in parallel.")
-      ->defaultValue(0);
+  schema_creator.addInt("ser_ref_levels", "Number of times to refine the mesh uniformly in serial.")->defaultValue(0);
+  schema_creator.addInt("par_ref_levels", "Number of times to refine the mesh uniformly in parallel.")->defaultValue(0);
 }
 
 }  // namespace mesh
 }  // namespace serac
+
+std::shared_ptr<mfem::ParMesh> FromInlet<std::shared_ptr<mfem::ParMesh>>::operator()(axom::inlet::Table& base)
```

Comment:
I was a bit confused at at first figuring out what this is doing. I know this really belongs in Axom, but renaming it `BuildFromInlet` might help.

Reply:
This is a good point. I've been mulling over the name of the `FromInlet` struct for a while and it's name is not as helpful as it could be.  `BuildFromInlet` is better for sure.

Reply:
I've got it! `TemplateCreateAndFillMagicStruct`

Reply:
I think either `Build` or `Construct` would be good. No comment on `MagicStruct`. 😄